bin/tessel-2.js

```diff
@@ -203,8 +203,13 @@ makeCommand('run')
     default: false,
     help: 'Deploy a project containing all files within, including those not used by the program, excluding any files matched by non-negated rules in .tesselignore and including any files matched by rules in .tesselinclude. Program is started from specified file.'
   })
+  .option('rustCC', {
```

Comment:
or possibly `rust-cc`? Unix command line options are almost always in kebab-case